SignalServiceKit/src/Messages/OWSMessageUtils.m

```diff
@@ -162,9 +199,11 @@ + (nullable TSQuotedMessage *)quotedMessageForMessage:(TSMessage *)message
         return nil;
     }
 
+    // It's conceivable that the logic above will find neither valid text
+    // or an attachment to quote.
```

Comment:
Outside of malformed data, I can't think of why this would happen. Should we assert?

Reply:
Done.